base/io.jl

```diff
@@ -645,3 +645,36 @@ ismarked(io::IO) = io.mark >= 0
 Commit all currently buffered writes to the given stream.
 """
 flush(io::IO) = nothing
+
+"""
+    skipchars(io::IO, predicate; linecomment=nothing)
+
+Skip forward in `io` until `predicate` returns `false`. If `linecomment`
+is defined, all characters after the `linecomment` character are ignored
+until the next line.
+
+```jldoctext
+julia> buf = IOBuffer("    text")
+IOBuffer(data=UInt8[...], readable=true, writable=false, seekable=true, append=false, size=8, maxsize=Inf, ptr=1, mar
+k=-1)
```

Comment:
I don't think wrapping the output mid-keyword would pass if you rand the doctests, would it?